tests/protractor/conf.js

```diff
@@ -74,6 +74,11 @@ const setupUser = () => {
     });
 };
 
+// From orbit, just to be sure
+process.on('exit', () => {
+  modules.forEach(module => module.kill());
```

Comment:
I had issues while testing where API would stay running sometimes on certain styles of catastrophic failure. This should force them to be killed if our process dies for whatever reason.